code/modules/mechs/components/arms.dm

```diff
@@ -44,3 +44,45 @@
 		to_chat(user, SPAN_NOTICE(" Actuator Integrity: <b>[round((((motivator.max_dam - motivator.total_dam) / motivator.max_dam)) * 100)]%</b>"))
 	else
 		to_chat(user, SPAN_WARNING(" Actuator Missing or Non-functional."))
+
+/obj/item/mech_component/manipulators/cheap
+	name = "lifter exosuit arms"
+	desc = "Industrial lifter arms that allow you to crudely manipulate things from the safety of your cockpit."
+	exosuit_desc_string = "industrial lifter arms"
+	icon_state = "loader_arms"
+	melee_damage = 15
```

Comment:
```suggestion
	melee_damage = WEAPON_FORCE_PAINFUL
```
You can use weapon damage defines if you want standardized damages (for other manipulators as well)

Reply:
I mean sure - but won't it look weird that only the power loader, and light arms are defined while the combat and heavy arms are not?

Reply:
Hmm, yeah. Maybe with some sort of `WEAPON_FORCE_LETHAL * multiplier` but it's not super clean.